src/components/templates/ListView/index.ts

```diff
@@ -120,7 +120,8 @@ class ListView extends BridgeBase {
     const theConfigProps = this.thePConn?.getConfigProps();
     if (theConfigProps) {
       this.selectionMode = theConfigProps.selectionMode;
-      const componentConfig = this.thePConn.getRawMetadata().config;
+      // @ts-ignore
```

Comment:
Can you leave an accurate description of the issue.  For example:

// @ts-ignore - required because ComponentMetadataConfig does not have config object in type def